test/e2e_node/startup_probe_test.go

```diff
@@ -58,15 +58,15 @@ var _ = framework.KubeDescribe("StartupProbe [Serial] [Disruptive] [NodeAlphaFea
 
 		/*
 			Release : v1.16
-			Testname: Pod liveness probe, using local file, delayed by startup probe
-			Description: A Pod is created with liveness probe that uses ‘exec’ command to cat the non-existent /tmp/health file. Liveness probe MUST NOT fail until startup probe expires.
+			Testname: Pod startup probe restart
+			Description: A Pod is created with a failing startup probe. The Pod MUST be killed and restarted incrementing restart count to 1, even if liveness would succeed.
 		*/
-		framework.ConformanceIt("should *not* be restarted with a exec \"cat /tmp/health\" because startup probe delays it [NodeConformance]", func() {
+		framework.ConformanceIt("should be restarted startup probe fails [NodeConformance]", func() {
```

Comment:
I'm very surprised these renames didn't require changing test/conformance/testdata/conformance.txt

@johnbelamaric, should this have touched that file?

Reply:
Hmm... looks like there are no files under test/e2e_node listed in conformance.txt. Are these actually considered conformance tests?

Reply:
Not sure... what's the requirement to consider them as conformance?